mailet/standard/src/main/java/org/apache/james/transport/mailets/LogLevel.java

```diff
@@ -0,0 +1,27 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.transport.mailets;
+
+
+public enum LogLevel {
+    DEBUG,
```

Comment:
Thinking about this we can enhance this rnum a lot by adding the LogWritter as a field:

```
public enum LogLevel {
    Debug((logger, message) -> logger.debug(message)),
    ...;

    private final LogWritter logWritter;

    // Constructor & getter
}
```

That way we can no more need LogMessage::selectLogWriter